src/factory.rs

```diff
@@ -397,6 +400,35 @@ impl Factory {
             target,
         }
     }
+
+    /// Create a basic mesh pipeline using a custom shader.
+    pub fn basic_pipeline(&mut self,
```

Comment:
I suppose this PR isn't finished yet? How can I use it in my program? We need an example.